src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -52,7 +53,7 @@ class SlackAlerter(BaseAlerter):
         slack_token: The Slack token tied to the Slack account to be used.
     """
 
-    slack_token: str
+    slack_token: str = SecretField()
```

Comment:
This is essentially a required field without any secret specified, so anyone creating a `SlackAlerter` object needs to pass in a `slack_token` otherwise it will fail.